### DISCUSSION TITLE
Pitch Count Not Displaying

### DISCUSSION THREAD

**Discussion by pjockey** [MLB-LED-Scoreboard/mlb-led-scoreboard#533] [Q&A] [answered]
I know this function was working last year, but I'm not seeing it this year. I don't see anything in the config.json and can't remember how to get it to work.

Can someone jog my memory, please and thanks?

Scott

**Comment by gotopes** [comment]
did you edit your matrix file (i.e. w64h32.json) to move the pitch count part to a coordinate that is off the matrix screen and need to re-edit it to return it to the screen?

**Reply by pjockey** [reply]
I edited the mlb-led-scoreboard/coordinates/w128x64.json.example, setting line 144 & 145 to true.

```
    "pitch_count": {
      "font_name": "4x6",
      "x": 1,
      "y": 50,
      "enabled": true,
      "append_pitcher_name": true
```
But, I guess I need to copy this file to `w128x64.json` as there is no file by that name in the directory.

**Reply by gotopes** [reply, marked as answer]
yes, you need your w128h64.json file in the coordinates directory and hopefully that helps.

**Reply by pjockey** [reply]
thanks for the help.
